src/Backend/Factory.php

```diff
@@ -56,7 +56,11 @@ public function buildRedisCache($options)
         }
 
         $redis = new \Redis();
-        $redis->connect($options['host'], $options['port'], $timeout);
+        if (empty($options['unix_socket'])) {
+            $redis->pconnect($options['host'], $options['port'], $timeout);
```

Comment:
Sorry, not related to this pull request, will remove.

https://github.com/piwik/plugin-QueuedTracking/pull/47#discussion_r87799311